R/parse.R

```diff
@@ -27,7 +27,7 @@ new_rcmdcheck <- function(stdout,
 
       rversion    = parse_rversion(entries),
       platform    = parse_platform(entries),
-      errors      = grep(" ...\\s+ERROR\n",   entries, value = TRUE),
+      errors      = grep("ERROR\n",   entries, value = TRUE),
```

Comment:
The new regex seems less restrictive than the original one. (I wonder what the purpose of the dots in the original regex is -- perhaps we need to write `"[.][.][.]"` or `"\\.\\.\\."` instead?)

Reply:
Yes, I also had this question. I think we need to wait for an answer from Gabor here.
It works fine for "Warnings" and "Notes" but "Errors" are sometimes not catched with the restrictive regex.

Maybe we now catch false positives due to the less restrictive regex but I haven't faced any yet.

Reply:
The new regex should be fine.

Reply:
Actually, not fine, because it matches the final status line, but I already fixed it.